.template/addons/svgeez/Gemfile.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# SVG Sprite
+insert_into_file 'Gemfile', after: /gem 'danger'.*\n/ do
+  <<~RUBY
+
+    # Svgeez
+    gem 'svgeez' # Gem for generating an SVG sprite from a folder of SVG icons.
+    gem 'inline_svg' # Use Inline SVG for styling SVG with CSS
```

Comment:
sorry if the [previous comment](https://github.com/nimblehq/rails-templates/pull/445/files#r1269099739) was a bit unclear,  `svgeez` is needed only during development ✅ but `inline_svg` requires to be in production as we'll be using `inline_svg_tag` inside the application.

Reply:
In a word:
-  `gem 'svgeez'`  -  development group
-   `gem 'inline_svg'` -  production group(the way you implemented in https://github.com/nimblehq/rails-templates/pull/445/commits/c0734c04a9e477ba39580d1811217dbbc4c432b3)

Reply:
Thanks. Fixed in 3b2dd797edfa8d0efad4e54a6c458f178e1c859a